.github/workflows/ci.yml

```diff
@@ -23,3 +23,25 @@ jobs:
         npm test
       env:
         CI: true
+
+  lint:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: npm install, build, and lint
+      run: |
+        npm ci
+        npm start
+        npm run lint
+      env:
+        CI: true
+        GAPI_MAX_PARALLEL: 3
```

Comment:
> Should we just run this on pull requests?

Actually, per [Pull request events for forked repositories](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#pull-request-events-for-forked-repositories), `push` events are not triggered in fork contexts, so I think we'll at least want these running on `pull_request`.

> Running it as a part of git-hook using [husky](https://www.npmjs.com/package/husky) before committing/pushing, maybe?

The linting is relatively taxing; I don't think we're going to want to be running it locally but happy to chat about it more.

Reply:
> `push` events are not triggered in fork contexts, so I think we'll

I can only see this about **pull** events, not **push**: "no pull request events occur on the forked repository". Am I missing something?...

> The linting is relatively taxing

Totally agree here, no need to enforce local linting for now

Reply:
> I can only see this about **pull** events, not **push**: "no pull request events occur on the forked repository". Am I missing something?...

Well, the fact that the `push` workflows have not been triggered within the context of _this_ pull request is telling. I moved the linting job to `pull_request`, so 🤞 that starts to work once this is merged to `master`?

Reply:
That did the trick! ✅